### DISCUSSION TITLE
Maybe someone has the original dataset 3q0907530aa ?

### DISCUSSION THREAD

**Discussion by cmaukr** [jilleb/MQB-FPA#67] [Exploration and research]
Maybe someone has the original dataset 3q0907530aa?

TNKS

**Comment by olli991** [comment]
Without naming the actual car this is worthless information

**Comment by JJordiJ** [comment]
I actually need this dataset as well for my VW Golf with retrofitted 3Q gw. SW is 4368. I’m willing to pay/contribute for it as well :)

**Reply by Mex77** [reply]
You can use any that has A*..
Just Google it..
